src/mgmt/xmgmt-root.c

```diff
@@ -11,6 +11,7 @@
 #include <linux/module.h>
 #include <linux/pci.h>
 #include <linux/aer.h>
+#include <linux/vmalloc.h>
```

Comment:
what is this header for? i don't see any code change...

Reply:
Needed to compile on POWER9 :-)